issuer-front-end/src/SelectNetwork.tsx

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-console */
 /* eslint-disable no-alert */
 import React, { ChangeEvent, Dispatch, SetStateAction, useEffect } from 'react';
+import { MODULE_REFERENCE_CREDENTIAL_REGISTRY } from './constants';
```

Comment:
There is a flow issue with the progress buttons now. If I go from step 1 to step 2 (without selecting a network there) then realize I made a mistake and go back to step 1. Now I am trapped because the `next` button to step 2 is disabled.

Another point:
When you are in step 4 and want to go back, you need to click twice the `previous` button before it jumps back to step 2. It is ok to jump back to step 2 but please at the first click to the `previous` button.

Another point:
Since the order of the steps were changed, this error text needs to be updated:
```
Alternatively, if you intend to use testnet, go back to step 1 and switch the network button.
Alternatively, if you intend to use mainnet, go back to step 1 and switch the network button.
```

Another point:
The `next` button should be disabled on the last page (step 4). People could thing the flow continues.

Reply:
All good points. I messed up your carefully designed steps :P Sorry about that.